admin/support.pl

```diff
@@ -330,6 +342,27 @@ sub collect_evidence
 		cp("/etc/systemd/system/nmis9d.service", "$targetdir/system_status/init");
 	}
 
+	my @daemons = ("nmis9d", "apache2", "httpd", "mongod"); # Add or remove daemons as needed
+
+	open(my $daemon_info_fh, '>>', "$targetdir/system_status/init/daemon_status") or die "Could not open file '$targetdir/system_status/init/daemon_status' $!";
+
+        foreach my $daemon (@daemons) {
+                my $status = `systemctl is-active $daemon 2>/dev/null`;
+                chomp $status;
```

Comment:
this should check to see if $status is something before it starts working with it